appveyor.yml

```diff
@@ -75,5 +78,8 @@ for:
 
     build_script:
       - ./bootstrap
-      - ./configure -q
-      - make -s
+      - ./configure
+      - make -j
```

Comment:
really `-j`? Will that blow up the AppVeyor VM?

Reply:
I guess there is only one way to find out...

Reply:
It worked. I won't wait for serial `make` unless there's no alternative.